### PR TITLE
remove remove_bid_after_accept setting

### PR DESCRIPTION
**Pull request by exciler**
As discussed on discord, bids should always be removed after filing a PIREP

### CODE REVIEW

File: app/Database/migrations/2021_03_25_213017_remove_setting_removebidonaccept.php [hunk 1]
```diff
@@ -0,0 +1,40 @@
+<?php
+
+use App\Services\Installer\SeederService;
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Support\Facades\DB;
+
+class RemoveSettingRemovebidonaccept extends Migration
```
Comment (nabeelio) [inline app/Database/migrations/2021_03_25_213017_remove_setting_removebidonaccept.php:7]: Nit, just rename/capitalize to `RemoveSettingBidOnAccept`

File: app/Database/migrations/2021_03_25_213017_remove_setting_removebidonaccept.php [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    }
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        $this->seederSvc->syncAllSettings();
```
Comment (nabeelio) [inline app/Database/migrations/2021_03_25_213017_remove_setting_removebidonaccept.php:38]: I don't think you need this

**Comment by nabeelio** [comment]
Minor comments, looks good to me!

**Comment by FatihKoz** [comment]
Just curious here ... Why are we removing the bid on pirep file ? I think it should be removed after pirep accept, even without a setting for it we need to keep the bid until accepting a pirep.

What happens when a pilots bids, starts a flight (pirep filed = bid removed), an error happens (sim crash/pc crash etc), he has to restart and does not want to resume previous flight. His (old) filed pirep will be deleted by phpvms, and bid was removed by us, so he needs to bid again.

Also when we remove the bid upon pirep file, this means that the same flight will be available for other pilots to bid, while the first bids holder is still flying the leg ;)

If we keep the removal at pirep accepted, i think all will be much nicer 'cause a pirep may be rejected automatically or manually, in this situations bid must be preserved.

**Comment by exciler** [comment]
Thank you @FatihKoz for bringing this up. Maybe we moved too fast :-)

But in my understanding, filing a PIREP in this context means, that we submit the PIREP. So we are referring to the point in time just before the PIREP is accepted or rejected. The pilot should have already be landed and a sim/pc crash would no longer effect this PIREP. I think the term "filed" is not precise enough.

Can you explain further, why a bid should be preserved in case the pirep is rejected? This would obviously affect out decision...

**Comment by FatihKoz** [comment]
Hi,

Actually I mixed **PreFile** with **File** :)

You are right, when a pirep gets filed this means that the user is arrived to destination and the flight is ended, but not **submitted** yet, at least this is what i understood from the functions. 

When i first looked the code, it thought it will be a problem if we remove the bid but now I agree with you ;) We can remove the bid at this point, if the pirep gets rejected the pilot has to start over and bid again.

File: app/Database/migrations/2021_03_25_213017_remove_setting_removebidonaccept.php (hunk 2 above)
Comment (exciler) [reply app/Database/migrations/2021_03_25_213017_remove_setting_removebidonaccept.php:38]: Yeah, was thinking about someone reverting to the old codebase... but then the migration is missing too... will remove it :-)

**Comment by exciler** [comment]
maybe we should rename the event from PirepFiled to PirepSubmitted as this seems more appropriate

**Comment by nabeelio** [comment]
I think it's mostly referred to as being "filed" in other places, I'll have to look. But I'll merge this one for now.

**Pull request merged**